Makefile.in

```diff
@@ -1,2428 +0,0 @@
-# Makefile.in generated by automake 1.16.1 from Makefile.am.
```

Comment:
Try to make minimal changes. In this case, it will be easier to port commits for the next version. If this file not needed for the build - ignore it, no need to remove.

Reply:
Ok, I followed the example of the previous version. Should I re-add them?

Reply:
> Ok, I followed the example of the previous version

History of the previous version:
- https://github.com/hunter-packages/libpng/commits/libpng16-hunter

Applied patches:
- https://github.com/hunter-packages/libpng/commit/2d58a3175172d709d60899c87f0ba143eeca691f
- https://github.com/hunter-packages/libpng/commit/525982359b07d6a3ccb76c9853ad29d1baedc1da
- https://github.com/hunter-packages/libpng/commit/e0c986be28eabf01bb8f3c01cae68a0cc332620b
- https://github.com/hunter-packages/libpng/commit/afce5a8a5cd3c1ecde2c1a23918feac96f6a8902
- https://github.com/hunter-packages/libpng/commit/df4bd7fe81219b4094ef7d03b10129633c7dd0e2

Nothing removed, mostly only CMake code touched.

> Should I re-add them?

Yes, please try to make minimal changes.